sdks/java/maven-archetypes/examples/pom.xml

```diff
@@ -43,9 +43,37 @@
       </extension>
     </extensions>
 
+
```

Comment:
Changes to `sdks/java/maven-archetypes/examples/pom.xml` and `sdks/java/maven-archetypes/examples-java8/pom.xml` appear identical.

Can we move them to their common parent at `sdks/java/maven-archetypes/pom.xml`?